scripts/attendance.py

```diff
@@ -0,0 +1,167 @@
+from calendar import Calendar, monthlen
+from datetime import datetime, timedelta
+DAILY_WORKING_HOURS = 9
+
+
+class AttendanceManagement():
+
+    def __init__(self):
+        self.today = datetime.now()
+        self.current_day = self.today.day
+        self.current_month = self.today.month
+        self.current_year = self.today.year
+
+    def time_to_minutes(self, time):
+        """
+        This accepts time format in put and convert it into minutes in integer
+         format
+        :param time:
+        :return: minutes in integer format
+        """
+
+        str_time = str(time)
+        time_list = str_time.split(":")
```

Comment:
What will happen if there is no ":" ?
What if there is no zeroth/first index ?
Need to add a try catch for IndexError or can we do it in a better way ?